### PR TITLE
603: adds addresses to api

### PR DESCRIPTION
**Pull request by glawson**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
+"use strict";
```
Comment (tabersays) [inline server/index.js]: This file should be ignored

File: server/index.js.map [hunk 2]
```diff
@@ -0,0 +1 @@
...AAC;YAC1B,MAAM,CAAC,GAAG,CAAC,MAAM,CAAC,GAAG,CAAC,CAAC,IAAI,CAAC;gBAC1B,KAAK,EAAE;oBACL,MAAM,EAAE,GAAG;oBACX,OAAO,EAAE,mBAAmB;iBAC7B;aACF,CAAC,CAAC;QACL,CAAC;QAED,SAAS,CAAC,SAAS,CAAC,GAAG,IAAI,CAAC;QAE5B,EAAE,CAAC,SAAS,CAAI,SAAS,yBAAsB,EAAE,IAAI,CAAC,SAAS,CAAC,SAAS,EAAE,IAAI,EAAE,CAAC,CAAC,EAAE,MAAM,EAAE,UAAC,KAAK;YACjG,EAAE,CAAC,CAAC,KAAK,CAAC,CAAC,CAAC;gBACV,MAAM,CAAC,GAAG,CAAC,MAAM,CAAC,GAAG,CAAC,CAAC,IAAI,CAAC;oBAC1B,KAAK,EAAE;wBACL,MAAM,EAAE,GAAG;wBACX,OAAO,EAAE,KAAK,CAAC,OAAO;qBACvB;iBACF,CAAC,CAAC;YACL,CAAC;YAED,MAAM,CAAC,GAAG,CAAC,IAAI,CAAC;gBACd,IAAI,aACF,GAAG,EAAE,SAAS,IACX,SAAS,CAAC,SAAS,CAAC,CACxB;aACF,CAAC,CAAC;QACL,CAAC,CAAC,CAAC;IACL,CAAC,CAAC,CAAC;AACL,CAAC,CAAC,CAAC;AAEH,GAAG,CAAC,GAAG,CAAC,OAAO,CAAC,MAAM,CAAC,IAAI,CAAC,IAAI,CAAC,SAAS,EAAE,SAAS,CAAC,CAAC,CAAC,CAAC;AAEzD,GAAG,CAAC,GAAG,CAAC,GAAG,EAAE,UAAC,GAAG,EAAE,GAAG;IACpB,GAAG,CAAC,QAAQ,CAAC,IAAI,CAAC,OAAO,CAAC,SAAS,EAAE,mBAAmB,CAAC,CAAC,CAAC;AAC7D,CAAC,CAAC,CAAC;AAGH,IAAM,MAAM,GAAG,GAAG,CAAC,MAAM,CAAC,OAAO,CAAC,GAAG,CAAC,IAAI,IAAI,IAAI,EAAE;IAClD,IAAM,IAAI,GAAG,MAAM,CAAC,OAAO,EAAE,CAAC,IAAI,CAAC;IAEnC,IAAM,SAAS,GAAG,uUAGI,CAAC;IAEvB,OAAO,CAAC,GAAG,CAAC,wDAAwD,EAAE,SAAS,EAAE,IAAI,CAAC,CAAC;AACzF,CAAC,CAAC,CAAC"}
```
Comment (tabersays) [inline server/index.js.map]: This file should be ignored

File: src/tsconfig.app.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
-  "extends": "../tsconfig.json",
```
Comment (tabersays) [inline src/tsconfig.app.json]: Put this back in, this is for the site and it is breaking tests

Comment (glawson) [reply src/tsconfig.app.json]: oops - wrong config, fixing

**Pull request merged**